src/App.xaml

```diff
@@ -16,6 +16,7 @@
                 <ResourceDictionary Source="/Styles/WindowTitleBar_ThemeResources.xaml" />
 
                 <!-- Resources from other projects -->
+                <ResourceDictionary Source="ms-appx:///DevHome.Dashboard/Styles/Dashboard_ThemeResources.xaml" />

```

Comment:
Thanks for continuing the pattern 🥳 I left a comment about the Environment in another PR to maintain a consistent structure.